deploy/README.md

```diff
@@ -0,0 +1,5 @@
+# 部署说明
```

Comment:
Roses are red, violets are blue, the repo is english so why aren't you? :notes:

Reply:
sorry , this part not need to merge here. I will close it